ksqldb-rest-app/src/main/java/io/confluent/ksql/api/server/ServerVerticle.java

```diff
@@ -233,7 +233,11 @@ private Router setupRouter() {
         .produces(KsqlMediaType.KSQL_V1_JSON.mediaType())
         .produces(JSON_CONTENT_TYPE)
         .handler(this::handleIsValidPropertyRequest);
-
+    router.route(HttpMethod.POST, "/test")
+        .handler(BodyHandler.create(false))
+        .produces(KsqlMediaType.KSQL_V1_JSON.mediaType())
+        .produces(JSON_CONTENT_TYPE)
```

Comment:
why do we need 2 JSON formats? shouldn't the `JSON_CONTENT_TYPE` format be enough? afaik the `KsqlMediaType.KSQL_V1_JSON` format is only used for pull/push queries.

Reply:
👍 removed `KsqlMediaType.KSQL_V1_JSON `